docs/diagrams/BetterModelClassDiagram.puml

```diff
@@ -17,5 +17,5 @@ Person -up-> "*" Tag
 Person *--> Name
 Person *--> Phone
 Person *--> Email
-Person *--> Address
+Person *--> RSVP
```

Comment:
Should the class name be `Rsvp` instead of `RSVP` according to the coding standard?